common-rest/src/main/java/org/uniprot/api/rest/output/converter/JsonMessageConverter.java

```diff
@@ -155,6 +155,11 @@ protected void after(MessageConverterContext<T> context, OutputStream outputStre
                 if (notNullNotEmpty(extraOptions.getSuggestedIds())) {
                     writeJsonArray(generator, "suggestedIds", extraOptions.getSuggestedIds());
                 }
+
+                if (Objects.nonNull(extraOptions.getObsoleteCount())) {
+                    generator.writeFieldName("obsoleteCount");
```

Comment:
what if we create a method for this like 'writeJsonObject'? (similar to writeJsonArray)